src/core/Microsoft.PowerToys.Settings.UI/Controls/HotkeySettingsControl.xaml

```diff
@@ -8,7 +8,27 @@
     mc:Ignorable="d"
     d:DesignHeight="300"
     d:DesignWidth="400">
-    <TextBox x:Name="HotkeyTextBox" 
-             IsReadOnly="True"
-             Header="{x:Bind Header, Mode=OneTime}"/>
-</UserControl>
+    <StackPanel Orientation="Vertical">
+        <ToolTipService.ToolTip>
+            <ToolTip>
+                <StackPanel Orientation="Vertical">
+                    <TextBlock x:Uid="ShortcutWarningLabel"/>
+                    <TextBlock Text="Win, Ctrl, Alt, Shift" FontWeight="SemiBold"/>

```

Comment:
Text Value (Text="Win, Ctrl, Alt, Shift") should be dynamic since we are going to use the control in multiple places.

Reply:
Good call. What would you recommend? A public string object would suffice, right?

Reply:
We can use an attribute and the value can be set in the XAML tag. Just like you did with the FontFamiliy attribute

Reply:
Yep, that's part of the latest commit.

Reply:
We can use an attribute and set the value in the xaml tag. Just like you did for FontFamily!